client/my-sites/plans-features-main/components/plan-notice.tsx

```diff
@@ -106,6 +122,32 @@ export default function PlanNotice( props: PlanNoticeProps ) {
 					) }
 				</Notice>
 			);
+		case PLAN_LEGACY_STORAGE_NOTICE:
+			return (
+				<Notice
+					className="plan-features-main__notice"
+					showDismiss={ true }
```

Comment:
Please, rebase your PRs before merging to trunk. Because #90499 landed before merging this PR, this line was introducing an ESLint error. I've resolved this in #90822.

Reply:
Actually, it's possible that the check didn't actually fail correctly, too.

Should be resolved by https://github.com/Automattic/wp-calypso/pull/90877 and checks will now fail as expected when there are new errors.

Reply:
Hi @tyxla thanks for letting me know.

I'll be sure to rebase before merging in the future 🙏